packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -111,16 +112,40 @@ const isWithinAvailableHours = (
   timeSlot: { start: ConfigType; end: ConfigType },
   {
     workingHours,
+    organizerTimeZone,
+    inviteeTimeZone,
   }: {
     workingHours: WorkingHours[];
+    organizerTimeZone: string;
+    inviteeTimeZone: string;
   }
 ) => {
   const timeSlotStart = dayjs(timeSlot.start).utc();
   const timeSlotEnd = dayjs(timeSlot.end).utc();
+  const isOrganizerInDST = isInDST(dayjs().tz(organizerTimeZone));
+  const isInviteeInDST = isInDST(dayjs().tz(organizerTimeZone));
+  const isOrganizerInDSTWhenSlotStart = isInDST(timeSlotStart.tz(organizerTimeZone));
+  const isInviteeInDSTWhenSlotStart = isInDST(timeSlotStart.tz(inviteeTimeZone));
+  const organizerDSTDifference = getDSTDifference(organizerTimeZone);
+  const inviteeDSTDifference = getDSTDifference(inviteeTimeZone);
+const sameDSTUsers = isOrganizerInDSTWhenSlotStart === isInviteeInDSTWhenSlotStart;
+const organizerDST = isOrganizerInDST === isOrganizerInDSTWhenSlotStart;
+const inviteeDST = isInviteeInDST === isInviteeInDSTWheSlotStart;
+  const getTime = (slotTime: Dayjs, minutes: number) =>
+    slotTime
+      .startOf("day")
+      .add(
+        sameDSTUsers && organizerDST && inviteeDST
+          ? minutes
+          : minutes -
+              (isOrganizerInDSTWhenSlotStart || isOrganizerInDST
+                ? organizerDSTDifference
+                : -inviteeDSTDifference),
```

Comment:
```suggestion
                : 0),
```

So this would fix my issue with `Could not book the meeting. No available users found.`. However, not sure if this would mess up something else?

Reply:
no, the subtraction sign should be removed from there